docs/Encapsulation.md

```diff
@@ -68,8 +66,7 @@ fastify.register(async function publicContext (childServer) {
     handler (request, response) {
       response.send({
         answer: request.answer,
-        foo: request.foo,
-        bar: request.bar
+        foo: request.foo
```

Comment:
Similarly, this is to show that `request.foo` is now available and will not be `undefined`, but `request.bar` _will_ still be `undefined`.